app/policies/user_context.rb

```diff
@@ -0,0 +1,8 @@
+class UserContext
+  attr_reader :request, :params
+
+  def initialize(request, params)
+    @request = request
+    @params = params
+  end
```

Comment:
@hsong-rh You might want to move the access object here

```
attr_reader :access_object

def access_object
    @access_object ||= Insights::API::Common::RBAC::Access.new.process
end

```

By doing it this way you will have access to the access_object in the resolve method of ApplicationPolicy and its sub classes since they all have the user object.

Reply:
👍 I will change this